pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/Resources.java

```diff
@@ -37,7 +36,7 @@ private Resources() {
         return new HashSet<>(
                 Arrays.asList(
                         FunctionApiV2Resource.class,
-                        WorkerStats.class,
+                        WorkerApiV2Resource.class,
```

Comment:
why do we name it v2? we don't have v1 for functions ?

Reply:
/worker is strictly v2. I see no point in adding a v1.